aspnetcore/grpc/authn-and-authz.md

```diff
@@ -101,13 +101,16 @@ public Ticketer.TicketerClient CreateClientWithCert(
 
 ### Other authentication mechanisms
 
-In addition to bearer token and client certificate authentication, all ASP.NET Core supported authentication mechanisms such as OAuth, OpenID and Negotiate should work with gRPC. Visit [ASP.NET Core authentication](xref:security/authentication/identity) for more information for configuring authentication on the server side.
+In addition to bearer token and client certificate authentication, most ASP.NET Core supported authentication mechanisms such as OAuth and OpenID should work with gRPC. Visit [ASP.NET Core authentication](xref:security/authentication/identity) for more information for configuring authentication on the server side.
```

Comment:
Can the word "should" be removed in that first sentence? It doesn't exactly instill confidence in the reader.

Reply:
I'm not confident because I'm not sure how many actually work with gRPC 😋 

Yeah it can be removed. That uncertainty is now covered by "most"

Reply:
Oh please don't. Most is a shitty way to do this. List what works.

Reply:
Most will work with gRPC. I'd rather have a list of authentication mechanisms that don't work. The first in that list being Windows auth.

Reply:
should removed.

Reply:
Sure, mention Windows auth definitely doesn't work, but don't be wishywashy about what does. Be definite in both.

Reply:
I've included a list. Let me know if there are common authN options I've missed, or out of date options in the list that should be removed.